crates/q_cli/src/cli/chat/mod.rs

```diff
@@ -96,6 +104,7 @@ const WELCOME_TEXT: &str = color_print::cstr! {"
 • Help me understand my git status
 
 <em>/acceptall</em>    <black!>Toggles acceptance prompting for the session.</black!>
+<em>/ed</em>           <black!>Open your $EDITOR to compose a prompt</black!>
```

Comment:
Can you remove /ed from the masthead text, but keep it in the /help text?

Reply:
Absolutely! Updated!